exchange/bidder.go

```diff
@@ -321,6 +329,11 @@ func (bidder *bidderAdapter) doRequest(ctx context.Context, req *adapters.Reques
 	}
 	httpReq.Header = req.Headers
 
+	// If adapter connection metrics are not disabled, add the client trace
+	// to get complete connection info into our metrics
+	if !bidder.me.disabledMetrics.AccountAdapterDetails {
+		ctx = bidder.addClientTrace(ctx)
```

Comment:
As I mentioned above, since the connection metrics can be enabled/disabled for all adapters, it might be easier to just add the http tracer earlier in the code path a i.e. somewhere before `requestBid` is called. That way you'd have to add the http tracer only once because the same context gets propagated downstream.

Reply:
The idea, as understood in the discussions we had on this PR, is to be able to enable this metrics by adapter.